test/e2e/e2e_suite_test.go

```diff
@@ -76,6 +77,8 @@ var _ = Describe("[Upgrade][Restic] Velero upgrade tests on cluster using the pl
 
 var _ = Describe("[Upgrade][Snapshot] Velero upgrade tests on cluster using the plugin provider for object storage and snapshots for volume backups", BackupUpgradeRestoreWithSnapshots)
 
+var _ = Describe("[backup-resource][deletion][restic] Velero tests on cluster using the plugin provider for object storage and Restic for volume backups", Backup_deletion_with_restic)
```

Comment:
Correct the case description

Reply:
Changing `[backup-resource][deletion]` to `[backup-deletion]` is more reasonable?

Reply:
There will be more tests under backup-resource, the resource does not mean as k8s resources, it is the resource caused by backup action, it's the object of velero.

Reply:
Should we use unified naming conventions like [BackupResource][Deletion][Restic]?